test/inductor/test_torchinductor.py

```diff
@@ -10850,6 +10850,28 @@ def fn(x, x2):
         _, code = run_and_get_code(fn, x, x2)
         FileCheck().check("aten.view.dtype(reinterpret_tensor").run(code[0])
 
+    @requires_gpu()
+    def test_scalar_cpu_tensor_arg(self):
+        def fn(x, y):
+            return x + y.sum()
+
+        test_dtypes = [
+            torch.float32,
+            torch.float64,
+            torch.float16,
+            torch.bfloat16,
+        ]
+        for cpu_dtype in test_dtypes:
+            x = torch.rand([20], device="cuda")
```

Comment:
Since you use `@requires_gpu()` this case will also run on Intel GPU(XPU), so I suggest use general `GPU_TYPE` to conver both cuda and xpu.
```suggestion
            x = torch.rand([20], device=GPU_TYPE)
```

Reply:
Hii, @FindHao  Can you also update this case so that we don't meet `CUDA is unavailable` in XPU device ?

Reply:
done https://github.com/pytorch/pytorch/pull/132702/commits/fad3c8eb757ba19f8511d6739d8c9dd63add129c